x/delegation/keeper/delegation_state.go

```diff
@@ -373,6 +373,27 @@ func (k *Keeper) GetAssociatedOperator(ctx sdk.Context, stakerID string) (string
 	return "", nil
 }
 
+func (k *Keeper) GetAssociatedStakers(ctx sdk.Context, operator string) ([]string, error) {
+	if _, err := sdk.AccAddressFromBech32(operator); err != nil {
+		return nil, delegationtype.ErrOperatorAddrIsNotAccAddr
+	}
+	store := prefix.NewStore(ctx.KVStore(k.storeKey), delegationtype.KeyPrefixAssociatedOperatorByStaker)
+	iterator := sdk.KVStorePrefixIterator(store, []byte{})
+	defer iterator.Close()
+
+	// assuming that we support 5 client chains, this is a reasonable capacity.
```

Comment:
Fair point. We don't need this restriction. But would you still advise a different starting capacity?

Reply:
No, I’m just not sure where this RPC will be used or whether it will impact the caller if multiple stakers from one client chain associate with a single operator.